.prettierrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  trailingComma: 'es5',
+  tabWidth: 2,
+  semi: false,
```

Comment:
I am open to debate about this but fr why would you put them when they aren't necessary